src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderTests.cs

```diff
@@ -304,5 +304,67 @@ public void Dispose()
                 }
             }
         }
+
+        [Fact]
+        public static void VerifyHashSignature_SHA256_2048()
```

Comment:
This is calling the legacy VerifyHash method on RSACryptoServiceProvider, the other one is calling the new VerifyHash method on RSA.

@ianhays Since the other methods are covered in RSACryptoServiceProviderBackCompat.cs, please add this there.  Something like new SignHash to old VerifyHash, and vice versa.  Doesn't need to be the big known value test case, as long as we test in terms of something else already being tested.